commit 084e3202637b3072ae26be448d1e8ce3ef1637f9 (refs/changes/65/5165/4)
Author: Robert Mustacchi <rm@joyent.com>
Date:   2018-12-01T16:28:14+00:00 (10 months ago)
    
    OS-7412 Need idnkit for g11n
    Reviewed by: Joshua M. Clulow <jmc@joyent.com>
    Reviewed by: John Levon <john.levon@joyent.com>
    Approved by: Joshua M. Clulow <jmc@joyent.com>

diff --git a/Makefile b/Makefile
index 8ac93cd..3f7db58 100644
--- a/Makefile
+++ b/Makefile
@@ -22,6 +22,12 @@ include $(CURDIR)/../../build.env
 BASE =		$(CURDIR)
 DESTDIR =	$(BASE)/proto
 
+#
+# This contains items that we only build during the strap build itself.
+#
+STRAP_ONLY = \
+	idnkit
+
 ifeq ($(STRAP),strap)
 
 STRAPPROTO =	$(DESTDIR)
@@ -41,7 +47,8 @@ SUBDIRS = \
 	nss-nspr \
 	openssl1x \
 	perl \
-	$(EXTRA_COMPILERS)
+	$(EXTRA_COMPILERS) \
+	$(STRAP_ONLY)
 
 STRAPFIX +=	$(PRIMARY_COMPILER) $(EXTRA_COMPILERS)
 STRAPFIX_SUBDIRS=$(STRAPFIX:%=%.strapfix)
@@ -224,7 +231,7 @@ install: $(PRIMARY_COMPILER) $(SUBDIRS)
 endif
 
 clean:
-	-for dir in $(PRIMARY_COMPILER) $(SUBDIRS) binutils; \
+	-for dir in $(PRIMARY_COMPILER) $(SUBDIRS) $(STRAP_ONLY) binutils; \
 	    do (cd $$dir; $(MAKE) DESTDIR=$(DESTDIR) clean); done
 	-rm -rf proto
 
diff --git a/idnkit/Makefile b/idnkit/Makefile
new file mode 100644
index 0000000..74043d8
--- /dev/null
+++ b/idnkit/Makefile
@@ -0,0 +1,24 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2018 Joyent, Inc.
+#
+
+VER = idnkit-2.3
+
+include ../Makefile.defs
+
+TARBALL =		$(VER).tar.bz2
+TARBALL_COMPRESS =	-j
+
+include ../Makefile.targ
+include ../Makefile.targ.autoconf
diff --git a/idnkit/idnkit-2.3.tar.bz2 b/idnkit/idnkit-2.3.tar.bz2
new file mode 100644
index 0000000..73b176e
Binary files /dev/null and b/idnkit/idnkit-2.3.tar.bz2 differ
